agent/statemanager/state_manager.go

```diff
@@ -47,7 +47,8 @@ const (
 	//   a) Refactor 'Internal' field in 'api.Container' to 'Type' enum
 	//   b) Add 'ContainerResourcesProvisioned' as a new 'ContainerStatus' enum
 	//   c) Add 'SteadyStateStatus' field to 'Container' struct
-	ECSDataVersion = 6
+	// 7) Add 'ENIAttachments' struct
+	ECSDataVersion = 7
```

Comment:
I don't think this needs to be 7. The current version in master is `5`. Please revert this to be `6` as I'd already incremented it in a previous commit to this branch.